source/configuration_manual/howto/rootless.rst

```diff
@@ -48,6 +48,25 @@ example configuration file exists in
 ``~/dovecot/share/doc/dovecot/example-config/`` and needs to be copied
 to ``~/dovecot/etc/dovecot/``.
 
+Add capabilities
+----------------
+
+Modern linux systems support capabilities which allows you to permit
+selective rights to processes. This allows you to run dovecot rootless
+without losing chroot and privileged ports.
+
+Use following commands to enable this
+
+.. code:: bash
+
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/submission-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/pop3-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/imap-login
+  setcap cap_sys_chroot,cap_net_bind_service+eip ~/dovecot/libexec/dovecot/lmtp
```

Comment:
But binding is done by dovecot master process, shouldn't it be the one getting cap_net_bind_service?

Reply:
i tested that this works.

Reply:
I don't understand why though. Does it then also fail if cap_net_bind_service is not set for these?

Reply:
Fixed.